resteasy-jaxrs/src/main/java/org/jboss/resteasy/core/ServerResponseWriter.java

```diff
@@ -336,12 +339,24 @@ public static void commitHeaders(BuiltResponse jaxrsResponse, HttpResponse respo
    {
       double q = -1;

```

Comment:
This SortableMediaType looks like the result of the S() function of spec section 3.7.2. If so both "q" and "qs" default value should be set to 1 :

> q and qs are assumed to be 1:0 if absent.